### ISSUE TITLE
LinkableTrend trait makes graph dissapear

### ISSUE THREAD

**Issue by rodrigoreyes79** [saintsystems/nova-linkable-metrics#11]
![Captura de Pantalla 2019-09-04 a la(s) 15 31 16](https://user-images.githubusercontent.com/3924426/64289021-0cfc9780-cf29-11e9-93ab-5da602e6de95.png)

**Package version:** 0.6
**Nova Version:** 2
**Laravel Version:** 5.8

Code:

```
class ContactsPerDay extends Trend
{
    use LinkableTrend;

    /**
     * Calculate the value of the metric.
     *
     * @param \Illuminate\Http\Request $request
     * @return mixed
     */
    public function calculate(Request $request)
    {
        return $this->countByDays($request, Contact::class);
    }

    /**
     * Get the ranges available for the metric.
     *
     * @return array
     */
    public function ranges()
    {
        return [
            30 => '30 Days',
            60 => '60 Days',
            90 => '90 Days',
        ];
    }

    /**
     * Determine for how many minutes the metric should be cached.
     *
     * @return  \DateTimeInterface|\DateInterval|float|int
     */
    public function cacheFor()
    {
        // return now()->addMinutes(5);
    }

    /**
     * Get the URI key for the metric.
     *
     * @return string
     */
    public function uriKey()
    {
        return 'contacts-per-day';
    }
}
```

**Comment by anderly** [comment]
@rodrigoreyes79 Can you tell me the specific version of Nova 2.x you're running. There are some changes in 2.1 and later that may have broken this and I haven't yet updated the package to be compatible.

**Comment by rodrigoreyes79** [comment]
@anderly From what I can see, it is Nova v2.1.1.

**Comment by anderly** [comment]
@rodrigoreyes79 this should be fixed by 74021843824e16621c83b9d831e49b13dc12a65e

**Comment by anderly** [comment]
We just published an update (0.1.0) in commit 5d8a765 that brings 3.x compatibility. Please update and let me know if the problem persists.
